tests/_testerlib.livecodescript

```diff
@@ -138,6 +188,14 @@ function TesterGetPrettyTestName pFilename
    end if
 end TesterGetPrettyTestName
 
+-- Prettify a test name by removing a ".compilertest" suffix
+function TesterGetPrettyCompilerTestName pFilename
+   if pFilename ends with ".compilertest" then
+      set the itemDelimiter to "."
+      return item 1 to -2 of pFileName
+   end if
+end TesterGetPrettyCompilerTestName
```

Comment:
Rather than adding this function, I suggest that you amend `TesterGetPrettyTestName()` so that it takes the suffix to remove as an argument.